webpack/plugins.ts

```diff
@@ -14,6 +14,9 @@ export const getMainPlugins = (outputDir: string, target: Target): any[] => [
     verbose: true
   }),
   new ProgressPlugin(),
+  new optimize.LimitChunkCountPlugin({
+    maxChunks: 1
+  }),
```

Comment:
Chrome MV3 broke without this. As webpack tried dynamic chunk loading with `document.createElement('script')` mechanism. So, told webpack bundle everything into single file.
I haven't used webpack since long, but I think there must be some mechanism to dynamic load? Single chunk works fine too anyway.